### ISSUE TITLE
Error when building iOS apps if using Android and iOS: Cannot read property 'resources' of null

### ISSUE THREAD

**Issue by ben-8409** [blakgeek/cordova-plugin-app-name#6]
With cordova 7.0 and when using and Android and iOS, i get the following error:

> Error: Cannot read property 'resources' of null

The problem is, that in rename-app.js, when reading the strings, "data" is not defined.
A simple check helps fixing the build on iOS:
```javascript
        parser.parseString(stringsXml, function (err, data) {
            if(!data || !data.resources) return;

            data.resources.string.forEach(function (string) {

                if (string.$.name === 'app_name') {

                    console.log('Setting App Name: ', name);
                    string._ = name;
                }
            });

            fs.writeFile(stringsPath, builder.buildObject(data));

        });
```

**Comment by firepol** [comment]
Thank you for the fix! I'm having the same issues and I'm still using cordova 6.2.0. I hope this fix is implemented soon.

**Comment by ben-8409** [comment]
My fix above was only a work-around, looking an the script, it seems that the script is in fact only required on android, but because of the current config.xml, the hook is run on iOS as well.
The following change to the config.xml should fix the issue.

`
    <preference name="APP_NAME"/>
    <hook type="after_prepare" src="src/rename-app.js"/>

    <platform name="android">
        <config-file target="config.xml" parent="/*">
            <preference name="AppName" value="$APP_NAME"/>
        </config-file>
    </platform>
`

To:
`
    <preference name="APP_NAME"/>

    <platform name="android">
        <config-file target="config.xml" parent="/*">
            <preference name="AppName" value="$APP_NAME"/>
        </config-file>
        <hook type="after_prepare" src="src/rename-app.js"/>
    </platform>
`

**Comment by blakgeek** [comment]
This has been fixed in version 1.0.4
